setup.py

```diff
@@ -88,8 +88,6 @@ def run(self):
 
 
 setup_args['extras_require'] = {
-    # azuread is required for use of AzureADOAuthenticator
-    'azuread': ['pyjwt>=2'],
```

Comment:
Is removing this entirely going to cause a error for users of this, foe example tljh? Does it require  a major release?